www/js/util.js

```diff
@@ -87,15 +87,14 @@ function formatUserlistItem(div) {
         profile: div.data("profile") || { image: "", text: ""},
         leader: div.data("leader") || false,
         icon: div.data("icon") || false,
-        afk: div.data("afk") || false
     };
     var name = $(div.children()[1]);
     name.removeClass();
     name.css("font-style", "");
     name.addClass(getNameColor(data.rank));
     div.find(".profile-box").remove();
 
-    if (data.afk) {
+    if (div.data("meta") && div.data().meta.afk) {
```

Comment:
was trying to be consistent actually, with the other lines in the block. I'm not sure why they are even pre-checking for the existence of `div.data("meta")`, not sure what situation would cause it to not exist